app/processor/string.go

```diff
@@ -69,84 +73,63 @@ func NewJobStringProcessor(
 	}
 }
 
-func getOption(options ...*Options) Options {
-	option := Options{Encoding: Plain}
-	for _, o := range options {
-		if o == nil {
-			continue
-		}
-		option.Encoding = o.Encoding
-		break
-	}
-	return option
-}
+func processOptionString(exp string) (opts Options) {
```

Comment:
this function does not work unfortunately. do you want me to fix it? (see private chat)